### ISSUE TITLE
Tab "Other" does not get updated after editing the biblatex source

### ISSUE THREAD

**Issue by koppor** [JabRef/jabref#3063]
Steps to reproduce:

1. Open an entry in the entry editor. The entry must have an other field such as `bibsource`
2. Switch to biblatex source
3. change `booktitle` to `mybooktitle`
4. Go to "Other fields"
5. "mybooktitle" does not appear as field

Refs #2002, #2064

Example entry:

```
@InProceedings{DBLP:conf/zeus/KoppLUW11,
  author      = {Oliver Kopp and Frank Leymann and Tobias Unger and Sebastian Wagner},
  title       = {Towards The Essential Flow Model},
  bibsource   = {dblp computer science bibliography, http://dblp.org},
  booktitle = {3rd Central-European Workshop on Services and their Composition, Services und ihre Komposition, {ZEUS} 2011, Karlsruhe, Germany, February 21-22, 2011. Proceedings},
}
```

**Comment by lenhard** [comment]
@koppor Can you please check if #3097 solves this problem for you?

**Comment by matthiasgeiger** [comment]
Fixed by #3097
